spec/article_json/import/google_doc/html/embedded_soundcloud_parser_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'embed_parser_shared'
+
+describe ArticleJSON::Import::GoogleDoc::HTML::EmbeddedSoundcloudParser do
+  include_context 'for an embed parser' do
+    let(:expected_embed_type) { :soundcloud }
+    let(:expected_embed_id) { 'rich-the-kid/plug-walk-1' }
+    let(:expected_tags) { %w(vimeo test) }
```

Comment:
`s/vimeo/soundcloud/`?

Reply:
good point,  or just test? what is it for anyway?

Reply:
Acutally the values don't really matter: the shared example writes the tags into the HTML fixture and then expects them to be extracted properly. Might still be less confusing to not use `vimeo` here ;)

Reply:
OK, I am going to do that. I guess I also have to add the support for AMP.

Reply:
I'm not entirely sure, but I think AMP should work out of the box